src/pages/ExplorerPage/ExplorerPage.tsx

```diff
@@ -35,42 +36,87 @@ const ExplorerPage = () => {
 
   const classes = useStyles();
   const [active, setActive] = useState(false);
+  // NOTE: we are going to use the setter upon filters implementation
   const [queryParams, setQueryParams] = useState({ page: "1", limit: "10" });
 
-  const [state, setState] = useState<{ transfers: Transfer[] | never[], loading: "none" | "loading" | "done", isReady: boolean }>({
+  const [state, setState] = useState<{
+    transfers: Transfer[] | never[];
+    loading: "none" | "loading" | "done";
+    isReady: boolean;
```

Comment:
Not too sure if both of these are needed? `done` state should only be used when all data has loaded.

Reply:
removed isReady